### PR TITLE
Update Dockerfile.ci

### PR DESCRIPTION
**Pull request by ayush-shah**
### Describe your changes:

Fixes <issue-number>

I worked on ... because ...

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: ingestion/Dockerfile.ci [hunk 1]
```diff
@@ -82,7 +82,7 @@ WORKDIR /home/airflow/openmetadata-airflow-apis
 RUN pip install "."
 
 WORKDIR /home/airflow/ingestion
-ARG INGESTION_DEPENDENCY="all"
+ARG INGESTION_DEPENDENCY
```
Comment (akash-jain-10) [inline ingestion/Dockerfile.ci:85]: @ayush-shah Why do we want to remove the default value of argument here ?

Comment (ayush-shah) [reply ingestion/Dockerfile.ci:85]: so if i modify env `INGESTION_DEPENDENCY` in the docker-compose file under ingestion container service:
it doesn't take that into account

Comment (akash-jain-10) [reply ingestion/Dockerfile.ci:85]: This is the argument / environment variable you need to pass during Docker build phase and not during the running the docker compose!

Comment (pmbrull) [reply ingestion/Dockerfile.ci:85]: I think that param is working as expected there. We should close this

**Pull request closed without merging**